### DISCUSSION TITLE
Public Roadmap

### DISCUSSION THREAD

**Discussion by csoare7** [lightdash/lightdash#1552] [General]
Hey guys, been following the project for a while now! It's really shaping up 💪 ⭐  
Do you have a public roadmap we can follow?

**Comment by TuringLovesDeathMetal** [comment]
Hey @csoare7! Sorry for the late reply - we were at an offsite all last week, so a bit unplugged 🧘‍♀️

Short answer: We used to, but we don't anymore. 

Long answer: We're still in really early stages as a company (only 11 months in!) and we found that our roadmap kept becoming pretty irrelevant, pretty quickly. It's hard to know what we're going to be building in the next 6 weeks, let alone the next 6 months. 

So, we ditched the long-term product roadmap. (We actually wrote a [blog post about it here](https://www.lightdash.com/blogpost/how-we-do-product-planning-at-lightdash)) 

But, we make a conscious effort to keep everything we're working on in GitHub and we build things that our users want/need! So you can scroll through some of the issues - if something is missing, please add it in :)

**Would be interested to know though: if we had a public roadmap, what would you use it for?**
